client/app/scripts/superdesk-archive/directives.js

```diff
@@ -243,6 +243,15 @@
                             familyService.fetchDesks(scope.item, false)
                                 .then(function(desks) {
                                     scope.desks = desks;
+                                    scope.tooltipText = '';
+
+                                    for (var i = 1; i < desks.length; i++) {
```

Comment:
Oh, OK then. It's probably good to add a comment, though, to explain that this is intentional.